vita3k/modules/SceAudio/SceAudio.cpp

```diff
@@ -164,8 +169,23 @@ EXPORT(int, sceAudioOutSetPortVolume_forUser) {
     return UNIMPLEMENTED();
 }
 
-EXPORT(int, sceAudioOutSetVolume) {
-    return UNIMPLEMENTED();
+EXPORT(int, sceAudioOutSetVolume, int port, SceAudioOutChannelFlag ch, int *vol) {
+    if (!ch) //no channel selected, no changes
+        return 0;
+
+    const AudioOutPortPtr prt = lock_and_find(port, host.audio.shared.out_ports, host.audio.shared.mutex);
+
+    const int left = (ch & SCE_AUDIO_VOLUME_FLAG_L_CH) ? vol[0] : prt->left_channel_volume;
+    const int right = (ch & SCE_AUDIO_VOLUME_FLAG_R_CH) ? vol[1] : prt->right_channel_volume;
```

Comment:
I'm not sure about this line, maybe make a note in case we encounter regression

Reply:
Reason is maybe if only R_CH is available, volumes array size is only 1? Im not sure, but we have not seen this yet